.github/workflows/common-java-maven-build.yml

```diff
@@ -0,0 +1,38 @@
+# This is a basic build workflow for Octopus Java modules published to Sonatype Central Repository
+name: Snapshot Compile&UT
+
+on:
+  workflow_call:
+    inputs:
+      java-version:
+        required: false
+        type: string
+        default: '8'
+    secrets:
+      OSSRH_USERNAME:
+        required: true
+      OSSRH_TOKEN:
+        required: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    environment: Prod
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Maven Central Repository
+        uses: actions/setup-java@v3
+        with:
+          java-version: ${{ inputs.java-version }}
+          distribution: 'adopt'
+          server-id: ossrh
+          server-username: MAVEN_USERNAME
+          server-password: MAVEN_PASSWORD
+      - name: Build with Maven
+        run: mvn --batch-mode --update-snapshots package
```

Comment:
May be Paranoia, but I preffere `clean` command before `package` - to get rid of possibe garbage left anywhere around from previous builds. Suggestion:
`mvn --batch-mode --update-snapshots clean package`

Reply:
It's okay to change. Make a pull request